idaes/power_generation/carbon_capture/compression_system/compressor.py

```diff
@@ -0,0 +1,411 @@
+
+        
+        self.kappaT = Var(self.flowsheet().config.time,
+                            initialize=5.0e-7)
+        self.c0 = Var(self.flowsheet().config.time,
+                        initialize = 260,
+                        doc="Speed of sound in the inlet gas stream",
+                        bounds=(0,4000)) # m/s
+        self.mass_flow_coeff = Var(self.flowsheet().config.time,
+                            initialize=0.0735,
+                            doc="Compressor flow coefficient",
+                            bounds=(0.01,0.15))        
+        self.I = Var(self.flowsheet().config.time,
+                            initialize=0.6993,
+                            doc="Impeller work coefficient",
+                            bounds=(0.001,1))
+        self.mu_p_v = Var(self.flowsheet().config.time,
+                            initialize = 0.60,
+                            doc="Polytropic head coefficient vaned diffuser",
+                            bounds=(0.001,1))        
+        self.mu_p = Var(self.flowsheet().config.time,
+                            initialize = 0.60,
+                            doc="Polytropic head coefficient",
+                            bounds=(0.001,1))
+        self.eff_p_v = Var(self.flowsheet().config.time,
+                            initialize=0.85,
+                            doc="Polytropic stage efficiency vaned diffuser",
+                            bounds=(0.001,1))
+        self.eff_p = Var(self.flowsheet().config.time,
+                            initialize=0.85,
+                            doc="Polytropic stage efficiency",
+                            bounds = (0.001, 1.0))       
+        
+        self.delta_enth_polytropic = Var(self.flowsheet().config.time,
+                            initialize=2548.5,
+                            doc="Polytropic enthalpy change [J/mol]",
+                            within=NonNegativeReals)
+        
+        self.U2 = Var(self.flowsheet().config.time,
+                            initialize = 315, 
+                            doc="Impeller tip speed", 
+                            bounds= (0,4000)) # upper: 4000, lower: 0 // m/s
+        self.Ma = Var(self.flowsheet().config.time,
+                            initialize = 0.1,
+                            doc = 'Rotational Ma number',
+                            bounds = (0,5))
+        self.rspeed = Var(self.flowsheet().config.time,
+                            initialize = 1500,
+                            doc = 'Rotation speed of the impeller',
+                            bounds = (0.5, 8000))      
+        self.r2 = Var(initialize= 0.075, 
+                            doc="Impeller tip radius")  
+        
+        ########################################################     
+        
+        self.dp = Param(default= 1, within=Reals, doc='a pressure difference for finite difference derivative calculation')
+        self.diffuser_type = Param(default=1.0, 
```

Comment:
These look like things that would be better defined as Config arguments, where users could use a named Enum rather than an integer to set these.

Reply:
Those have been modified in the revised version.